temp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Could you please move this ignore to that in root folder? From time to time, I remove all temp folder so it might be inconvenient.